test/integration/targets/dnf/tasks/dnf.yml

```diff
@@ -456,6 +456,32 @@
 - name: check bar with rpm
   command: rpm -q bar
   register: bar_result
+# Newer Fedora Docker images come with coreutils-single which is incompatible
+# with coreutils (required by @Web Server). We force the install of coreutils
+# before running the environment group install.
+# https://github.com/fedora-cloud/docker-brew-fedora/issues/58
+- name: ensure coreutils is installed over coreutils-single
+  dnf:
+    name: coreutils
+    state: present
+    allowerasing: yes
+  changed_when: '"Nothing to do" not in coreutils_install.msg'
+  register: coreutils_install
+
+- block:
+  - name: install Web Server environment
+    dnf:
+      name: "@Web Server"
```

Comment:
*sigh* It looks like this group is not available on RHEL 8. I feel like we should try and create some kind of dummy group for testing purposes.

Reply:
Ah, cool, we already have one - https://github.com/ansible/ansible/blob/devel/test/integration/targets/setup_rpm_repo/files/comps.xml.

Reply:
Yea I think this is just a merge conflict issue. We originally did the coreutils removal with allow-erasing using a shell command. Since then I created that custom group and removed the `Web Server` one.

We should still have a test for allowerasing just not like this.

Reply:
@mkrizek @jborean93 yeah, I think I goobered this up on the merge conflict resolve .... will work on it.

Reply:
@jborean93 thought on the tests? I thought about it a bit more but honestly if this doesn't fail then we're in good shape, I can't think of a consistent scenario there we'd need to force `allowerasing` for a test case but if you have ideas I'm on board with more/better tests :)

Reply:
It's been a while since I last looked at this but we should create a custom package that has a conflicted dependency and will only continue with `allowerasing`. This shouldn't be too hard and because it's our mocked rpm we are not at the mercy of distro changes over time.